app/views/devise/two_factor_authentication/show_totp.html.slim

```diff
@@ -8,7 +8,7 @@ p
   '<strong>#{APP_NAME}</strong>.
 = form_tag([:user, :two_factor_authentication], method: :put, role: 'form') do
   .mb2
-    = label_tag 'code', raw('<abbr title="required">*</abbr> ') + t('forms.two_factor.code')
+    = label_tag 'code', t('forms.two_factor.code')
```

Comment:
same as above. we need to include `<abbr title="required">*</abbr>` markup with required fields